### ISSUE TITLE
Cannot start Winpinator

### ISSUE THREAD

**Issue by michalderlatka** [swiszczoo/winpinator#19]
On both W10 and W11 I'm not able to run winpinator

![image](https://user-images.githubusercontent.com/2479155/164984765-8df9460e-c33b-41cc-98c7-b04b37608d1e.png)

**Comment by swiszczoo** [comment]
Hi, can you check if this file exists: `%APPDATA%\Winpinator\winpinator.lock`? It should be created when Winpinator starts. If it doesn't exist, can you create an empty one and check if it will be removed? I have no clue what's going on but that might help to debug it. And I just want to ensure, do you have any free disk space?

**Comment by michalderlatka** [comment]
There is a lock file. I have a plenty of free disk space. 
I checked few more things
1) I am able to start application running as administartor
2) I uninstalled app. Then installed again and it started without any issues for the first time. After closing and starting again the same issue as above happened.

Tested both scenarios on both computers (W10 & W11) with exactly the same result.

**Comment by michalderlatka** [comment]
Looks like I found what might be wrong. When you close application with X button it doesn't really close. Process is still runnning, so it cannot be started once again. After killing process in task manager I'm able to start app once again.

I think it would be good to add some kind of notification (winpinator is still running) in tray. With collapsed tray you cannot see it is still running.

**Comment by ghost** [comment]
> Process is still running

Yes, and that should be checked by the user by disabling that utterly annoying non-feature of hiding systray icons that the system enables by default. One might be surprised by what still runs in the background without their knowledge.

That said, I believe each application should either give up this toxic practice of mapping the Close button to tray minimize, or offer the user the option to choose between minimize-to-tray and straight-up close in their Settings panel.
If the author chooses the second variant and if minimize-to-tray is enabled by default then yes, there should be a one-time notification about the state of the application (at first run, when "closed" to tray), together with a mention of the ability to change that behavior.

**Comment by swiszczoo** [comment]
Yeah, but that's not how it was supposed to work. When you started Winpinator for the second time, it successfully detected that another process is already running, so it tried to tell that older instance to restore a window by some kind of IPC (DDE, to be exact) but somehow failed. I can't think of any reason why it doesn't work on your computer. I thought that maybe the older process was running under privilleged account (by having a selected checkbox in the installer) and the new one under normal account and that broke something, but I just can't replicate it.

**Comment by michalderlatka** [comment]
After few days problem suddenly disappeared. I am now able to start Winpinator for the second time and this restores older instance's window.
